libs/runtime/ebpf_hash_table.c

```diff
@@ -965,7 +972,20 @@ ebpf_hash_table_next_key(
 size_t
 ebpf_hash_table_key_count(_In_ const ebpf_hash_table_t* hash_table)
```

Comment:
nit: maybe we can add a comment for this function that this function is expensive to call in hot path when `EBPF_HASH_TABLE_NO_LIMIT` is set, just to ensure someone does not use it inadvertently .